web_widget_html_markdown/readme/USAGE.rst

```diff
@@ -0,0 +1,8 @@
+Your XML form view definition should contain::
+
+    ...
+    <field name="field_name" widget="html_markdown"/
+    ...
+
+
+This will render the Html field as typical html and edit as markdown or HTML.
```

Comment:
... "This will replace the default Html widget by the new Markdown/HTML widget and allow the field to be edited as Markdown or HTML, depending on the user's choice."